cmake/external/mkldnn.cmake

```diff
@@ -19,7 +19,7 @@ SET(MKLDNN_PREFIX_DIR    ${THIRD_PARTY_PATH}/mkldnn)
 SET(MKLDNN_INSTALL_DIR    ${THIRD_PARTY_PATH}/install/mkldnn)
 SET(MKLDNN_INC_DIR        "${MKLDNN_INSTALL_DIR}/include" CACHE PATH "mkldnn include directory." FORCE)
 SET(MKLDNN_REPOSITORY     https://github.com/intel/mkl-dnn.git)
```

Comment:
can you have two target in MKLDNN repository, one can generate mkldnn.lib&mkldnn.dll, another can generate dnnl.lib&dnnl.dll. Then windows will use mkldnn.lib&mkldnn.dll

Reply:
I don't choose library name inside our CMake, it's done in mkl-dnn build files so it's not possible.